libcontainer/configs/intelrdt.go

```diff
@@ -1,6 +1,9 @@
 package configs
 
 type IntelRdt struct {
+	// The identity for RDT Class of Service
```

Comment:
Perhaps you need to document it more, for example, say that
 * if not set, it equals the container ID;
 * if explicitly set, it is assumed that the corresponding resctrl directory is managed externally, and it is not removed by runc when he container is destroyed.

Reply:
Or, if this is sufficiently documented in runtime-spec, please disregard the above comment.

Reply:
K